### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by sungam3r**
Hey. I saw this project in tests for Castle.Core and got interested. Looking through the code I noticed several typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ public void my_assembly_has_no_public_api_changes()
 }
 ```
 
-### Shoudly
+### Shouldly
```
Comment (danielmarbach) [inline README.md]: Should be Shoudly

Other than that looks good to me!

**Comment by danielmarbach** [comment]
Thx for the contribution!

File: README.md (hunk 1 above)
Comment (sungam3r) [reply README.md]: https://github.com/shouldly/shouldly
изображение

Comment (danielmarbach) [reply README.md]: Ahaha that happens when you inherit a project. Thanks for educating me 😁

**Pull request merged**